storage/client/proposals.go

```diff
@@ -0,0 +1,98 @@
+package client
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+
+	governanceV22 "github.com/oasisprotocol/nexus/coreapi/v22.2.11/governance/api"
+	registryV22 "github.com/oasisprotocol/nexus/coreapi/v22.2.11/registry/api"
+	roothashV22 "github.com/oasisprotocol/nexus/coreapi/v22.2.11/roothash/api"
+	schedulerV22 "github.com/oasisprotocol/nexus/coreapi/v22.2.11/scheduler/api"
+	stakingV22 "github.com/oasisprotocol/nexus/coreapi/v22.2.11/staking/api"
+	governanceV23 "github.com/oasisprotocol/nexus/coreapi/v23.0/governance/api"
+	keymanagerV23 "github.com/oasisprotocol/nexus/coreapi/v23.0/keymanager/api"
```

Comment:
Just checking - is `keymanagerV22` not present because the struct doesn't exist in damask?

Also, do we need to include the cobalt types here? [convert_tx.go](https://github.com/oasisprotocol/nexus/blob/main/analyzer/consensus/convert_tx.go) might be a helpful reference.

Reply:
There's no such types for Cobalt, since `ChangeParameters` proposals were only introduced with Damask. 

But v24 does introduce a new consensus app - Vault, which also has `ChangeProposals`. We need to add v24 types. I opened: https://app.clickup.com/t/8694wjwh2

v24 is generally non-breaking (only adds stuff) so I think we should be able to add the new stuff into the v23.0?

Reply:
Done in https://github.com/oasisprotocol/nexus/pull/716 will add `vault.ChangeParameters` here, once that PR is merged and this one is rebased.

Reply:
Added